recipes/pysirius/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "0.9" %}
+{% set name = "py-sirius-ms" %}
+{% set commit = "5064de70794ce0b28ace80f33e536c7a374c56f6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/boecker-lab/sirius-client-openAPI/archive/{{ commit }}.zip
+  sha256: 84f19f6a48ea510c040642c5e81e2a1b3e3a0a1fe0823900241ea797306a04b0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install ./client-api_python/generated -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+
+  run:
+    - python >=3.8
+    - certifi >=14.05.14
+    - six >=1.10
+    - python-dateutil >=2.5.3
+    - setuptools >=21.0.0
+    - urllib3 >=1.15.1
+    - sirius-ms
+
+test:
+  imports:
+    - PySirius
+  requires:
+    - pip
+  commands:
+    - pip check
+    - sh $RECIPE_DIR/test_script.sh           # [not win]
```

Comment:
This script relies on Java, right? I am assuming it is installed because sirius depends on it, but it doesn't hurt to be explicit about the deps you need in your tests.